ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java

```diff
@@ -197,6 +197,12 @@ public BlockProcessingResult processBlock(
       }
     }
 
+    final ValidatorExitsValidator exitsValidator = protocolSpec.getExitsValidator();
+    if (exitsValidator instanceof ValidatorExitsValidator.AllowedExits) {
```

Comment:
I think that the verification to perform the popping should belong to the validator/processor itself and not the blockprocessor. Maybe an extra method allowExists()?